webviz_subsurface/plugins/_seismic_misfit.py

```diff
@@ -3497,26 +3508,31 @@ def make_polygon_df(ensemble_set: dict, polygon: str) -> pd.DataFrame:
             else:
                 for poly_file in poly_files:
                     logging.debug(f"Read polygon file:\n {poly_file}")
-                    df_polygon = pd.read_csv(
-                        poly_file, usecols=["POLY_ID", "X_UTME", "Y_UTMN"]
-                    )
+                    df_polygon = pd.read_csv(poly_file)
                     cols = df_polygon.columns
-                    if (
-                        ("POLY_ID" not in cols)
-                        or ("X_UTME" not in cols)
-                        or ("Y_UTMN" not in cols)
+
+                    if ("ID" in cols) and ("X" in cols) and ("Y" in cols):
+                        df_polygon = df_polygon[["X", "Y", "ID"]]
+                    elif (
+                        ("POLY_ID" in cols)
+                        and ("X_UTME" in cols)
+                        and ("Y_UTMN" in cols)
                     ):
+                        df_polygon = df_polygon[["X_UTME", "Y_UTMN", "POLY_ID"]].rename(
+                            columns={"X_UTME": "X", "Y_UTMN": "Y", "POLY_ID": "ID"}
+                        )
```

Comment:
Should we add a `UserWarning` here and say that the user should change from e.g. `X_UTME` to `X`? This would be a gentle push towards help following FMU (and future fmu-dataio/SUMO) standard.